_includes/graphql/classes.md

```diff
@@ -0,0 +1,81 @@
+# Classes
+
+Since your application does not have a schema yet, you can use the `createClass` mutation to create your first class through the **GraphQL API**. Run the following:
```

Comment:
```suggestion
Since your application does not have a schema yet, you can use the `createClass` mutation to create your first class through the GraphQL API. Run the following:
```